### DISCUSSION TITLE
The best trade-off of performance & quality.?

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2944] [Q&A] [answered]
There are lots of options in fooocus
LCM, Turbo Speed: very quick but poor quality when it's the rich prompt.
The default model: It seems to use steps and better quality but the performance is poor.

How can I configure the settings to ensure that the processing time stays under 15 seconds while also maintaining the quality of the images when using a detailed prompt?

**Comment by ILikeToasters** [comment, marked as answer]
Your speed is determined by your hardware and the settings. Quality setting will give you best results for photographic type images and speed will usually work well for other types like anime or comics for example. There is no magic way of doing quality and high speed generation unless you have better hardware. So I hate to say it but your answer is you probably need better hardware if you need high quality and fast image generation. Or pay for an online service that provides that.

**Reply by oldhand7** [reply]
The computer is VSP with GPU. 
Nvidia RTX 4100 , Memory: 8GB.

The current fooocus take 40s for generating 1 image right now.

I would like to see if it 's possible to get it within 15s with good quality.

**Reply by oldhand7** [reply]
I also tested the turbo speed mode and it's performance is quite good but when the prompt is getting complex, the quality degrades too much. Can anyone give me an advice to me?

**Comment by AnalyticsXXI** [comment]
For more complex prompt, you might try to increase "Guidance scale" on the "Advanced" tab. This way, it will be more in line with your prompt and maybe speed mode will be not so bad.
